app/src/main/java/de/qspool/clementineremote/ui/fragments/LibraryFragment.java

```diff
@@ -342,7 +359,12 @@ private boolean addSongsToPlaylist(MyLibraryItem libraryItem) {
             case MyLibrary.LVL_ARTIST:
                 MyLibrary addArtist = new MyLibrary(getActivity());
                 addArtist.addOnLibrarySelectFinishedListener(LibraryFragment.this);
-                addArtist.getAllTitlesFromArtistAsync(libraryItem.getArtist());
+                if(LibraryAlbumOrder.ALPHABET.equals(mAlbumOrder)) {
```

Comment:
If we have the overload of `getAlbum()` with `LibraryAlbumOrder`, we don't need the if anymore